bandwidth-file-spec.txt

```diff
@@ -302,6 +323,118 @@
 
       This Line was added in version 1.3.0 of this specification.
 
+    "recent_consensus_count=" Int NL
+
+      [Zero or one time.].
+
+      The number of the different consensuses seen in the last data_period
+      days. (data_period is 5 by default.)
+
+      Assuming that Tor clients fetch a consensus every 1-2 hours,
+      and that the data_period is 5 days, this line SHOULD be between:
+          data_period * 24 / 2 =  60
+          data_period * 24     = 120
+
+      This Line was added in version 1.4.0 of this specification.
+
+    "recent_priority_list_count=" Int NL
+
+      [Zero or one time.]
+
+      The number of times that a list with a subset of relays prioritized
+      to be measured has been created in the last data_period days.
+      (data_period is 5 by default.)
+
+      In 2019, with 7000 relays in the network, this line SHOULD be between:
+      TODO
+
+      This Line was added in version 1.4.0 of this specification.
+
+    "recent_priority_relay_count=" Int NL
+
+      [Zero or one time.]
+
+      The number of relays that has been in in the list of relays prioritized
+      to be measured in the last data_period days. (data_period is 5 by
+      default.)
+
+      In 2019, with 7000 relays in the network, this line SHOULD be between:
+      TODO
+
+      This Line was added in version 1.4.0 of this specification.
+
+    "recent_measurement_attempt_count=" Int NL
+
+      [Zero or one time.]
+
+      The number of times that any relay has been queued to be measured
+      in the last data_period days. (data_period is 5 by default.)
+
+      In 2019, with 7000 relays in the network, this line SHOULD be between:
+      TODO
+
+      This Line was added in version 1.4.0 of this specification.
+
+    "recent_measurement_failure_count=" Int NL
+
+      [Zero or one time.]
+
+      The number of times that the scanner attempted to measure a relay in
+      the last data_period days (5 by default), but the pre-conditions
```

Comment:
Which are the pre-conditions?. 
I don't know how to express this, but this is not due an error (network failures, no second relay..), but probably a program bug.

Reply:
Changed in https://github.com/teor2345/torspec/pull/1